projects/landing-page/src/styles/variables.css

```diff
@@ -0,0 +1,49 @@
+:root {
```

Comment:
Just a copy paste from de voorhoede. Will be cleaned up when the design is ready